README.md

```diff
@@ -72,8 +73,7 @@ In order to use this template to publish on Maven Central, you need to configure
 | `ORG_GRADLE_PROJECT_SIGNING_KEY` | The GPG Private key to sign your artifacts. You can obtain it with `gpg --armor --export-secret-keys <your@email.here>` or you can create one key online on [pgpkeygen.com](https://pgpkeygen.com). The key starts with a `-----BEGIN PGP PRIVATE KEY BLOCK-----`. |
 | `ORG_GRADLE_PROJECT_SIGNING_PWD` | The passphrase to unlock your private key (you picked it when creating the key). |
 
-The template already sets up [Dokka](https://kotlin.github.io/dokka/) for project documentation and attaches `-sources.jar` to your publications,
```

Comment:
Removing Dokka is problematic as it breaks publishing to Maven Central so we should rather fix it instead of removing ti

Reply:
I haven't worked with Dokka before, I just noticed that it's mentioned in the `libs.versions.toml` file but doesn't actually appear to be referenced anywhere in the Gradle build config. Does it get used implicitly somehow? https://github.com/Kotlin/dokka suggests that it should be mentioned in the gradle files somewhere.

So I think I've just removed Dokka as a dependency in `libs.versions.toml` and updated the README.md to reflect this change. I think Dokka was already not being used in this template. Have I missed something?

Reply:
Thanks for pointing out. We're fine removing dokka as we moved to `withJavadocJar()` some time ago and I forgot to clean this up.